<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Argentum Online Fiuba</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="logo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>501</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/logo.png);
border:0px;</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>700</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/background.jpg);</string>
    </property>
    <property name="interactive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>410</y>
      <width>208</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/hover-button.9.png);</string>
    </property>
    <property name="text">
     <string>Conectar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="hostLineEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>300</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="servicelabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>350</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string>Service:</string>
    </property>
   </widget>
   <widget class="QLabel" name="hostlabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>300</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string>Host:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="serviceLineEdit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>350</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="conexionestadoLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>460</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Estado Conexion</string>
    </property>
   </widget>
   <widget class="QLabel" name="serverStateLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>490</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>logo</zorder>
   <zorder>connectButton</zorder>
   <zorder>hostLineEdit</zorder>
   <zorder>servicelabel</zorder>
   <zorder>hostlabel</zorder>
   <zorder>serviceLineEdit</zorder>
   <zorder>conexionestadoLabel</zorder>
   <zorder>serverStateLabel</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
